packages/gatsby/src/utils/worker/__tests__/share-state.ts

```diff
@@ -235,28 +235,9 @@ describe(`worker (share-state)`, () => {
       staticQueryID
     )
 
-    expect(components).toMatchInlineSnapshot(`
-      Object {
-        "Head": false,
-        "componentPath": "/foo",
-        "config": false,
-        "isInBootstrap": true,
-        "isSlice": false,
-        "pages": Object {},
-        "query": "I'm a page query",
-        "serverData": false,
-      }
-    `)
+    expect(components).toMatchInlineSnapshot(`null`)
```

Comment:
Are you sure that components should results with null instead of 
Object {
        "Head": false,
        "componentPath": "/foo",
        "config": false,
        "isInBootstrap": true,
        "isSlice": false,
        "pages": Object {},
        "query": "I'm a page query",
        "serverData": false,
      }